PayPal/src/main/java/com/braintreepayments/api/PayPalBrowserSwitchRequest.java

```diff
@@ -0,0 +1,17 @@
+package com.braintreepayments.api;
+
+/**
+ * Request parameters associated with a {@link PayPalPendingRequest.Started}
+ */
+public class PayPalBrowserSwitchRequest {
```

Comment:
Is this class helpful/necessary? It just wraps a `BrowserSwitchPendingRequest.Started`, which is already a public class for direct browser switch integrations. We could just have `PayPalPendingRequest.Started` have a `BrowserSwitchPendingRequest.Started` as a param instead. Thoughts?

Reply:
Only thing it does is wrap browser switch. Keeping browser switch encapsulated by the SDK may be its only responsibility at this point.

Reply:
Ok will keep it in 👍